src/deploy/utils.py

```
@@ -472,3 +496,70 @@ def check_pipedream_api_support() -> dict:
+            continue
+        kept.append(tok)
+
+    # 2-tuples, not the full 5-tuple form: untokenize with positions tries to
+    # preserve the original line/col of every surviving token and pads the gaps
+    # left by removed comments with backslash continuations — dozens of stray
+    # "\\" lines at the top of the payload. Dropping positions reconstructs
+    # clean source from the token text alone.
+    text = tokenize.untokenize([(t.type, t.string) for t in kept])
+    lines, cleaned, blanks = text.split("\n"), [], 0
+    for line in lines:
+        line = line.rstrip()
```

Ruff (0.16.1)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)